src/Glpi/RichText/UserMention.php

```diff
@@ -261,4 +266,46 @@ public static function refreshUserMentionsHtmlToDisplay(string $content): string
 
         return $content;
     }
+
+    public static function isEnabled(): bool
+    {
+        $pro = Profile::getById($_SESSION['glpiactiveprofile']['id']);
+        return $pro->fields['use_mentions'] !== self::USER_MENTION_DISABLED;
+    }
+
+    public static function isMentionsRestricted(): bool
+    {
+        $pro = Profile::getById($_SESSION['glpiactiveprofile']['id']);
+        return $pro->fields['use_mentions'] === self::USER_MENTION_RESTRICTED;
+    }
+
+    public static function getRestrictedUsers(CommonITILObject $item): array
+    {
+        $data = [
+            'enabled' => self::isEnabled(),
```

Comment:
It is preferable to always return the same `keys` in a JSON object. Also, defining them at only one place is easier to read.

```suggestion
            'enabled' => self::isEnabled(),
            'full' => !self::isMentionsRestricted(),
            'user' => [],
```